scripts/operations/node_management/ncn-image-modification.sh

```diff
@@ -624,7 +624,7 @@ fi
 set_timezone
 if [[ $CSM_RELEASE =~ ^1\.4\.4 ]]; then
   csm_144_patch
-elif [[ $CSM_RELEASE =~ ^1\.4\.([5-9]|[1-9]0+|[1-9]{2,}) ]]; then
+elif [[ $CSM_RELEASE =~ ^1\.4\.([1-9][0-9]+|[0-9]{2,}|[5-9]) ]]; then
```

Comment:
```suggestion
elif [[ $CSM_RELEASE =~ ^1\.4\.([5-9]|[1-9][0-9]+) ]]; then
```

Reply:
Should actually be `^1\.4\.([5-9]|[1-9]0+|[1-9]{2,})`.
Somehow I pasted the right one into the PR description and then undid/reverted to a hacked one in the actual commit.
Fixed.